src/api/controller/api/enrichment.js

```diff
@@ -99,6 +104,6 @@ app.use(koaBodyParser());
 app.use(route.post('/', postEnrichment));
 app.use(route.put('/:id', putEnrichment));
 app.use(route.delete('/:id', deleteEnrichment));
-app.use(route.post('/schedule/:action/:id', scheduleDatasetEnrichment));
+app.use(route.post('/background/:action/:id', enrichmentBackground));
```

Comment:
IMO it would be 'restier' to have a `/:id/:action` instead of `/background/:action/:id`